src/utilities/Utilities.js

```diff
@@ -2,6 +2,10 @@ class Utilities {
   static base64Encode (string) {
     return global.atob(string)
   }
+
+  static formatDate (date, tz, format) {
+    return '01/02/03 04:05'
```

Comment:
Is there a reason to not format passed `date` here, or you didn't want to complicate things?

Reply:
yes but that's only for now. I'm planning on its modification along with a Stub Configuration for OAuth Service. Not sure when though.